MapboxCoreNavigation/RouteController.swift

```diff
@@ -296,21 +296,34 @@ open class RouteController: NSObject {
      */
     public var location: CLLocation? {
         guard let location = rawLocation else { return nil }
-        guard let stepCoordinates = routeProgress.currentLegProgress.currentStep.coordinates else { return nil }
-        guard let snappedCoordinate = Polyline(stepCoordinates).closestCoordinate(to: location.coordinate) else { return location }
 
         var nearByCoordinates = routeProgress.currentLegProgress.nearbyCoordinates
-        let nearByPolyline = Polyline(nearByCoordinates)
 
         // If the upcoming maneuver a sharp turn, only look at the current step for snapping.
         // Otherwise, we may get false positives from nearby step coordinates
         if let upcomingStep = routeProgress.currentLegProgress.upComingStep,
             let initialHeading = upcomingStep.initialHeading,
             let finalHeading = upcomingStep.finalHeading,
-            initialHeading.differenceBetween(finalHeading) < RouteControllerMaxManipulatedCourseAngle,
             let coordinates = routeProgress.currentLegProgress.currentStep.coordinates {
-           nearByCoordinates = coordinates
+            
+            // Calculate if angle is sharp
+            let inAngle = initialHeading * .pi / 180.0
+            let outAngle = finalHeading * .pi / 180.0
```

Comment:
Turf provides `toDegrees()` and `toRadians()` methods for these conversions.

Reply:
Hmm when doing something like:

```swift
let inAngle: CLLocationDegrees = initialHeading.toRadians()
let outAngle: CLLocationDegrees = finalHeading.toRadians()
```

I get a `'toRadians' is inaccessible due to 'internal' protection level`. Ref implementation: https://github.com/mapbox/turf-swift/blob/c0651d438dde71519a0469956b69ef641573fcae/Turf/CoreLocation.swift#L21

Reply:
@bsudekum maybe the extension needs to be marked `public` or `open` as well?

Reply:
@bsudekum @frederoni maybe the extension itself needs to be marked `public` or `open` as well?

Reply:
Currently the function is private and not yet in a release. Fixing this with https://github.com/mapbox/turf-swift/pull/19